CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+tools, including links to documentation, source code and some information about where they are used.
+
+Apart from these tools, the repository includes:
+
+* Some documenation in the [docs](docs) directory.
+* A Python module called [osclib](osclib) which includes code that is shared by several tools. They
+  are available in the `osclib` package.
+* A Docker-based [tests suite](tests). The Docker manifests and the Docker Compose files are
+  located in the [dist](dist) directory.
+* [GoCD](https://www.gocd.org) configuration files in [gocd](gocd). GoCD is an open source CI/CD
+  server that it is used to deploy the bots on OBS.
+* A set of [Tampermonkey](https://www.tampermonkey.net) scripts (see [userscript](userscript)
+  directory) to extend OBS features when using the web interface.
+* Several [systemd](systemd) units, as many of the tools (e.g., the `staging-bot`) make use of
+  them.
+
+## Tools
+
+Most of these tools are available as packages for several distributions. Check the [spec file in
+this repository](dist/package/openSUSE-release-tools.spec) or the [devel
+project](https://build.opensuse.org/package/show/openSUSE:Tools/openSUSE-release-tools) for further
+information.
+
+For the time being, we have classified them into three different groups: *command line tools*, *OBS
+bots* and *osc plugins*. Bear in mind that the information in the following list might be wrong and
+incomplete.
+
+### Command Line Tools
+
+ Usually, the executables are renamed as `osrt-NAME` (e.g., `osrt-announcer`).
+
+#### announcer
+
+Generates email diffs summaries to announce product releases.
+
+* Sources: [factory-package-news/announcer.py](factory-package-news/announcer.py)
+* Documentation: [factory-package-news/README.asciidoc](factory-package-news/README.asciidoc)
+* Package: openSUSE-release-tools-announcer
+
+#### pkglistgen
+
+Generates and updates OBS products for openSUSE and SLE. It generates package lists based on
+`000package-groups` and puts them in `000product` (resulting kiwi files) and `000release-packages`
+(release package spec files).
+
+* Sources: [pkglistgen.py](pkglistgen.py)
+* Documentation: [docs/pkglistgen.md](docs/pkglistgen.md)
+* Package: openSUSE-release-tools-pkglistgen
+* Usage: gocd
+
+#### container_cleaner
+
+Cleans up the `maintenance_release` projects for containers
+
+* Sources: [container_cleaner.py](container_cleaner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### metrics
+
+Generates insightful metrics from relevant OBS and annotation data, based on InfluxDB and Grafana.
+
+* Sources: [metrics.py](metrics.py)
+* Documentation: [docs/metrics.md](./docs/metrics.md)
+* Package: openSUSE-release-tools-metrics
+* Usage: ?
+
+#### metrics-access
+
+Ingests `download.opensuse.org` Apache access logs and generate metrics. It is composed of a PHP.
+script and a set of [systemd units](systemd).
```

Comment:
is it single script or more of them as I see more of it in given directory...and also example config file.

Reply:
Yes, you are right, it is not a script but two :smiley:  I will rephrase it a little bit.